tools/scope_manage_annotation.py

```
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+import argparse
+import pandas as pd
+from scope.fritz import api
+
+
+def manage_annotation(action, source, group_ids, token, origin, key, value):
+
+    # Forgiving definitions of actions
+    if action in ['update', 'Update', 'UPDATE']:
+        action = 'update'
+    elif action in ['delete', 'Delete', 'DELETE']:
+        action = 'delete'
+    elif action in ['post', 'Post', 'POST']:
+        action = 'post'
+
+    # Initial checks for origin and key, always necessary
+    if origin is None:
+        print('Error: please specify origin to %s' % action)
+        return
+
+    if key is None:
+        print('Error: please specify key to %s' % action)
+        return
+
+    # check if source is single object or csv file of many
+    if source[-4:] == '.csv':
+        file = pd.read_csv(source)  # modify input formats to prepare for loop
+        obj_ids = file['obj_id']
```

Maybe gracefully throw error if this header missing?